### ISSUE TITLE
Lint 8.7 support

### ISSUE THREAD

**Issue by trevjonez** [slackhq/compose-lints#413]
Hot off the press with 8.7 we are catching a new error from lint.

```
Caused by: java.lang.RuntimeException: Unexpected failure during lint analysis (this is a bug in lint or one of the libraries it depends on) |  
-- | --
  |   |  
  | Message: Unexpected failure during lint analysis (this is a bug in lint or one of the libraries it depends on) |  
  |   |  
  | Message: Unexpected failure during lint analysis of Cards.kt (this is a bug in lint or one of the libraries it depends on) |  
  |   |  
  | Message: getText(...) must not be null |  
  |   |  
  | The crash seems to involve the detector \\\`slack.lint.compose.ParameterOrderDetector\\\`. |  
  | You can try disabling it with something like this: |  
  | android { |  
  | lint { |  
  | disable "ComposeParameterOrder" |  
  | } |  
  | } |  
  |   |  
  | Stack: \\\`NullPointerException:ParameterOrderDetector$Companion$createErrorMessage$1.invoke(ParameterOrderDetector.kt:30)←ParameterOrderDetector$Companion$createErrorMessage$1.invoke(ParameterOrderDetector.kt:30)←StringsKt__AppendableKt.appendElement(Appendable.kt:85)←CollectionsKt___CollectionsKt.joinTo(_Collections.kt:3490)←CollectionsKt___CollectionsKt.joinToString(_Collections.kt:3507)←CollectionsKt___CollectionsKt.joinToString$default(_Collections.kt:3506)←ParameterOrderDetector$Companion.createErrorMessage(ParameterOrderDetector.kt:30)←ParameterOrderDetector.visitComposable(ParameterOrderDetector.kt:98)←ComposableFunctionDetector$createUastHandler$1.visitMethod(ComposableFunctionDetector.kt:33)←UElementVisitor$DispatchUastVisitor.visitMethod(UElementVisitor.kt:535)←UElementVisitor$DelegatingUastVisitor.visitMethod(UElementVisitor.kt:818)←UMethod.accept(UMethod.kt:42)←ImplementationUtilsKt.acceptList(implementationUtils.kt:15)←AbstractKotlinUClass.accept(AbstractKotlinUClass.kt:213)←ImplementationUtilsKt.acceptList(implementationUtils.kt:15)←UFile.accept(UFile.kt:89)←UastLintUtilsKt.acceptSourceFile(UastLintUtils.kt:953)←UElementVisitor$visitFile$3.run(UElementVisitor.kt:216)←LintClient.runReadAction(LintClient.kt:1781)←LintDriver$LintClientWrapper.runReadAction(LintDriver.kt:2747)←UElementVisitor.visitFile(UElementVisitor.kt:213)←LintDriver$visitUastDetectors$1.run(LintDriver.kt:2048)←LintClient.runReadAction(LintClient.kt:1781)←LintDriver$LintClientWrapper.runReadAction(LintDriver.kt:2747)←LintDriver.visitUastDetectors(LintDriver.kt:2048)←LintDriver.visitUast(LintDriver.kt:2010)←LintDriver.runFileDetectors(LintDriver.kt:1284)←LintDriver.checkProject(LintDriver.kt:1070)←LintDriver.checkProjectRoot(LintDriver.kt:622)←LintDriver.access$checkProjectRoot(LintDriver.kt:174)←LintDriver$analyzeOnly$1.invoke(LintDriver.kt:445)←LintDriver$analyzeOnly$1.invoke(LintDriver.kt:442)←LintDriver.doAnalyze(LintDriver.kt:501)←LintDriver.analyzeOnly(LintDriver.kt:442)←LintCliClient$analyzeOnly$1.invoke(LintCliClient.kt:258)←LintCliClient$analyzeOnly$1.invoke(LintCliClient.kt:255)←LintCliClient.run(LintCliClient.kt:312)←LintCliClient.analyzeOnly(LintCliClient.kt:255)←Main.run(Main.java:1766)←Main.run(Main.java:282)←DirectMethodHandleAccessor.invoke(DirectMethodHandleAccessor.java:103)←Method.invoke(Method.java:580)←AndroidLintWorkAction.invokeLintMainRunMethod(AndroidLintWorkAction.kt:103)←AndroidLintWorkAction.runLint(AndroidLintWorkAction.kt:90)←AndroidLintWorkAction.execute(AndroidLintWorkAction.kt:64)←DefaultWorkerServer.execute(DefaultWorkerServer.java:63)←NoIsolationWorkerFactory$1$1.create(NoIsolationWorkerFactory.java:66)←NoIsolationWorkerFactory$1$1.create(NoIsolationWorkerFactory.java:62)←ClassLoaderUtils.executeInClassloader(ClassLoaderUtils.java:100)←NoIsolationWorkerFactory$1.lambda$execute$0(NoIsolationWorkerFactory.java:62)←AbstractWorker$1.call(AbstractWorker.java:44)←AbstractWorker$1.call(AbstractWorker.java:41)←DefaultBuildOperationRunner$CallableBuildOperationWorker.execute(DefaultBuildOperationRunner.java:209)←DefaultBuildOperationRunner$CallableBuildOperationWorker.execute(DefaultBuildOperationRunner.java:204)←DefaultBuildOperationRunner$2.execute(DefaultBuildOperationRunner.java:66)←DefaultBuildOperationRunner$2.execute(DefaultBuildOperationRunner.java:59)←DefaultBuildOperationRunner.execute(DefaultBuildOperationRunner.java:166)←DefaultBuildOperationRunner.execute(DefaultBuildOperationRunner.java:59)←DefaultBuildOperationRunner.call(DefaultBuildOperationRunner.java:53)←AbstractWorker.executeWrappedInBuildOperation(AbstractWorker.java:41)←NoIsolationWorkerFactory$1.execute(NoIsolationWorkerFactory.java:59)←DefaultWorkerExecutor.lambda$submitWork$0(DefaultWorkerExecutor.java:174)←FutureTask.run(FutureTask.java:317)←DefaultConditionalExecutionQueue$ExecutionRunner.runExecution(DefaultConditionalExecutionQueue.java:194)←DefaultConditionalExecutionQueue$ExecutionRunner.access$700(DefaultConditionalExecutionQueue.java:127)←DefaultConditionalExecutionQueue$ExecutionRunner$1.run(DefaultConditionalExecutionQueue.java:169)←Factories$1.create(Factories.java:31)←DefaultWorkerLeaseService.withLocks(DefaultWorkerLeaseService.java:263)←DefaultWorkerLeaseService.runAsWorkerThread(DefaultWorkerLeaseService.java:127)←DefaultWorkerLeaseService.runAsWorkerThread(DefaultWorkerLeaseService.java:132)←DefaultConditionalExecutionQueue$ExecutionRunner.runBatch(DefaultConditionalExecutionQueue.java:164)←DefaultConditionalExecutionQueue$ExecutionRunner.run(DefaultConditionalExecutionQueue.java:133)←Executors$RunnableAdapter.call(Executors.java:572)←FutureTask.run(FutureTask.java:317)←ExecutorPolicy$CatchAndRecordFailures.onExecute(ExecutorPolicy.java:64)←AbstractManagedExecutor$1.run(AbstractManagedExecutor.java:48)←ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1144)←ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:642)←Thread.run(Thread.java:1583)\\\` |  
  | Stack: \`RuntimeException:LintDriver$Companion.handleDetectorError(LintDriver.kt:3740)←LintDriver$Companion.handleDetectorError$default(LintDriver.kt:3607)←LintDriver$Companion.handleDetectorError(LintDriver.kt:3603)←UElementVisitor.visitFile(UElementVisitor.kt:242)←LintDriver$visitUastDetectors$1.run(LintDriver.kt:2048)←LintClient.runReadAction(LintClient.kt:1781)←LintDriver$LintClientWrapper.runReadAction(LintDriver.kt:2747)←LintDriver.visitUastDetectors(LintDriver.kt:2048)←LintDriver.visitUast(LintDriver.kt:2010)←LintDriver.runFileDetectors(LintDriver.kt:1284)←LintDriver.checkProject(LintDriver.kt:1070)←LintDriver.checkProjectRoot(LintDriver.kt:622)←LintDriver.access$checkProjectRoot(LintDriver.kt:174)←LintDriver$analyzeOnly$1.invoke(LintDriver.kt:445)←LintDriver$analyzeOnly$1.invoke(LintDriver.kt:442)←LintDriver.doAnalyze(LintDriver.kt:501)←LintDriver.analyzeOnly(LintDriver.kt:442)←LintCliClient$analyzeOnly$1.invoke(LintCliClient.kt:258)←LintCliClient$analyzeOnly$1.invoke(LintCliClient.kt:255)←LintCliClient.run(LintCliClient.kt:312)←LintCliClient.analyzeOnly(LintCliClient.kt:255)←Main.run(Main.java:1766)←Main.run(Main.java:282)←DirectMethodHandleAccessor.invoke(DirectMethodHandleAccessor.java:103)←Method.invoke(Method.java:580)←AndroidLintWorkAction.invokeLintMainRunMethod(AndroidLintWorkAction.kt:103)←AndroidLintWorkAction.runLint(AndroidLintWorkAction.kt:90)←AndroidLintWorkAction.execute(AndroidLintWorkAction.kt:64)←DefaultWorkerServer.execute(DefaultWorkerServer.java:63)←NoIsolationWorkerFactory$1$1.create(NoIsolationWorkerFactory.java:66)←NoIsolationWorkerFactory$1$1.create(NoIsolationWorkerFactory.java:62)←ClassLoaderUtils.executeInClassloader(ClassLoaderUtils.java:100)←NoIsolationWorkerFactory$1.lambda$execute$0(NoIsolationWorkerFactory.java:62)←AbstractWorker$1.call(AbstractWorker.java:44)←AbstractWorker$1.call(AbstractWorker.java:41)←DefaultBuildOperationRunner$CallableBuildOperationWorker.execute(DefaultBuildOperationRunner.java:209)←DefaultBuildOperationRunner$CallableBuildOperationWorker.execute(DefaultBuildOperationRunner.java:204)←DefaultBuildOperationRunner$2.execute(DefaultBuildOperationRunner.java:66)←DefaultBuildOperationRunner$2.execute(DefaultBuildOperationRunner.java:59)←DefaultBuildOperationRunner.execute(DefaultBuildOperationRunner.java:166)←DefaultBuildOperationRunner.execute(DefaultBuildOperationRunner.java:59)←DefaultBuildOperationRunner.call(DefaultBuildOperationRunner.java:53)←AbstractWorker.executeWrappedInBuildOperation(AbstractWorker.java:41)←NoIsolationWorkerFactory$1.execute(NoIsolationWorkerFactory.java:59)←DefaultWorkerExecutor.lambda$submitWork$0(DefaultWorkerExecutor.java:174)←FutureTask.run(FutureTask.java:317)←DefaultConditionalExecutionQueue$ExecutionRunner.runExecution(DefaultConditionalExecutionQueue.java:194)←DefaultConditionalExecutionQueue$ExecutionRunner.access$700(DefaultConditionalExecutionQueue.java:127)←DefaultConditionalExecutionQueue$ExecutionRunner$1.run(DefaultConditionalExecutionQueue.java:169)←Factories$1.create(Factories.java:31)←DefaultWorkerLeaseService.withLocks(DefaultWorkerLeaseService.java:263)←DefaultWorkerLeaseService.runAsWorkerThread(DefaultWorkerLeaseService.java:127)←DefaultWorkerLeaseService.runAsWorkerThread(DefaultWorkerLeaseService.java:132)←DefaultConditionalExecutionQueue$ExecutionRunner.runBatch(DefaultConditionalExecutionQueue.java:164)←DefaultConditionalExecutionQueue$ExecutionRunner.run(DefaultConditionalExecutionQueue.java:133)←Executors$RunnableAdapter.call(Executors.java:572)←FutureTask.run(FutureTask.java:317)←ExecutorPolicy$CatchAndRecordFailures.onExecute(ExecutorPolicy.java:64)←AbstractManagedExecutor$1.run(AbstractManagedExecutor.java:48)←ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1144)←ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:642)←Thread.run(Thread.java:1583)\` |  
  | Stack: `RuntimeException:LintDriver$Companion.handleDetectorError(LintDriver.kt:3740)←LintDriver$Companion.handleDetectorError$default(LintDriver.kt:3607)←LintDriver$Companion.handleDetectorError(LintDriver.kt:3603)←LintDriver$analyzeOnly$1.invoke(LintDriver.kt:447)←LintDriver$analyzeOnly$1.invoke(LintDriver.kt:442)←LintDriver.doAnalyze(LintDriver.kt:501)←LintDriver.analyzeOnly(LintDriver.kt:442)←LintCliClient$analyzeOnly$1.invoke(LintCliClient.kt:258)←LintCliClient$analyzeOnly$1.invoke(LintCliClient.kt:255)←LintCliClient.run(LintCliClient.kt:312)←LintCliClient.analyzeOnly(LintCliClient.kt:255)←Main.run(Main.java:1766)←Main.run(Main.java:282)←DirectMethodHandleAccessor.invoke(DirectMethodHandleAccessor.java:103)←Method.invoke(Method.java:580)←AndroidLintWorkAction.invokeLintMainRunMethod(AndroidLintWorkAction.kt:103)←AndroidLintWorkAction.runLint(AndroidLintWorkAction.kt:90)←AndroidLintWorkAction.execute(AndroidLintWorkAction.kt:64)←DefaultWorkerServer.execute(DefaultWorkerServer.java:63)←NoIsolationWorkerFactory$1$1.create(NoIsolationWorkerFactory.java:66)←NoIsolationWorkerFactory$1$1.create(NoIsolationWorkerFactory.java:62)←ClassLoaderUtils.executeInClassloader(ClassLoaderUtils.java:100)←NoIsolationWorkerFactory$1.lambda$execute$0(NoIsolationWorkerFactory.java:62)←AbstractWorker$1.call(AbstractWorker.java:44)←AbstractWorker$1.call(AbstractWorker.java:41)←DefaultBuildOperationRunner$CallableBuildOperationWorker.execute(DefaultBuildOperationRunner.java:209)←DefaultBuildOperationRunner$CallableBuildOperationWorker.execute(DefaultBuildOperationRunner.java:204)←DefaultBuildOperationRunner$2.execute(DefaultBuildOperationRunner.java:66)←DefaultBuildOperationRunner$2.execute(DefaultBuildOperationRunner.java:59)←DefaultBuildOperationRunner.execute(DefaultBuildOperationRunner.java:166)←DefaultBuildOperationRunner.execute(DefaultBuildOperationRunner.java:59)←DefaultBuildOperationRunner.call(DefaultBuildOperationRunner.java:53)←AbstractWorker.executeWrappedInBuildOperation(AbstractWorker.java:41)←NoIsolationWorkerFactory$1.execute(NoIsolationWorkerFactory.java:59)←DefaultWorkerExecutor.lambda$submitWork$0(DefaultWorkerExecutor.java:174)←FutureTask.run(FutureTask.java:317)←DefaultConditionalExecutionQueue$ExecutionRunner.runExecution(DefaultConditionalExecutionQueue.java:194)←DefaultConditionalExecutionQueue$ExecutionRunner.access$700(DefaultConditionalExecutionQueue.java:127)←DefaultConditionalExecutionQueue$ExecutionRunner$1.run(DefaultConditionalExecutionQueue.java:169)←Factories$1.create(Factories.java:31)←DefaultWorkerLeaseService.withLocks(DefaultWorkerLeaseService.java:263)←DefaultWorkerLeaseService.runAsWorkerThread(DefaultWorkerLeaseService.java:127)←DefaultWorkerLeaseService.runAsWorkerThread(DefaultWorkerLeaseService.java:132)←DefaultConditionalExecutionQueue$ExecutionRunner.runBatch(DefaultConditionalExecutionQueue.java:164)←DefaultConditionalExecutionQueue$ExecutionRunner.run(DefaultConditionalExecutionQueue.java:133)←Executors$RunnableAdapter.call(Executors.java:572)←FutureTask.run(FutureTask.java:317)←ExecutorPolicy$CatchAndRecordFailures.onExecute(ExecutorPolicy.java:64)←AbstractManagedExecutor$1.run(AbstractManagedExecutor.java:48)←ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1144)←ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:642)←Thread.run(Thread.java:1583)` |  
```

**Comment by ZacSweers** [comment]
Thanks for the report, we're gonna cut a new release soon. If you could test latest snapshots that'd be helpful to confirm this is fixed

**Comment by trevjonez** [comment]
Ahh you pushed 1.4 before I was able to get back to this. 1.4 is obviously working as you might expect. Thank you for getting on top of this so fast!
